p256/fiat_p256.mli

```diff
@@ -13,17 +24,16 @@ val pp_point_error : Format.formatter -> point_error -> unit
 (** Pretty printer for point parsing errors *)
 
 val point_of_cs : Cstruct.t -> (point, point_error) result
-(** Convert from cstruct. Refuses the point at infinity.
-    The format is the uncompressed format described in
-    SEC1, section 2.3.4, that is to say:
+(** Convert from a cstruct encoding a point in the uncompressed format
```

Comment:
I prefer function docstrings of the form `[point_of_cs buffer] converts [buffer] to a [point]. The [buffer] is expected to be uncompressed (see ...). The following checks are performed: ...` -- i.e. showing the call-site of that function and providing names for the arguments. (and again a missing `:` for the hyperlink).